components/org.wso2.carbon.identity.application.authenticator.adapter/src/main/java/org/wso2/carbon/identity/application/authenticator/adapter/internal/constant/AuthenticatorAdapterConstants.java

```diff
@@ -37,6 +40,7 @@ public class AuthenticatorAdapterConstants {
     public static final String LOCAL_IDP = "LOCAL";
     public static final String FED_IDP = "FEDERATED";
     public static final String EXECUTION_STATUS_PROP_NAME = "actionExecutionStatus";
+    public static final String MULTI_ATTR_SEPARATOR = FrameworkUtils.getMultiAttributeSeparator();
```

Comment:
Let's not use this call to setup a static value. This call performs some service calls underlying